src/Policy/PolicyInterface.php

```diff
@@ -6,10 +6,15 @@
 
 use Chubbyphp\Deserialization\Denormalizer\DenormalizerContextInterface;
 
+/**
+ * @method bool isCompliantIncludingPath(string $path, object $object, DenormalizerContextInterface $context)
+ */
 interface PolicyInterface
 {
     /**
-     * @param object $object
+     * @deprecated
      */
-    public function isCompliant(DenormalizerContextInterface $context, $object): bool;
+    public function isCompliant(DenormalizerContextInterface $context, object $object): bool;
```

Comment:
we shouldn't have adjusted the interface by adding the object typehint. This is a bc in my understanding. @dominikzogg  what do you think? Shall we do a bugfix release of the serializer?

Reply:
yes, please